symphony-bdk-core/src/main/java/com/symphony/bdk/core/util/AntPathMatcher.java

```diff
@@ -0,0 +1,965 @@
+package com.symphony.bdk.core.util;
+
+/*
+ * Copyright 2002-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Are we OK with the license?

Reply:
Yes, this is allowed, see : https://www.whitesourcesoftware.com/resources/blog/top-10-apache-license-questions-answered/#3_Is_the_Apache_considered_copyleft